presentations/2023-11-27-bsi-value-types.md

```diff
@@ -256,6 +256,60 @@ Both of the proposed types, `indirect` and `polymorphic`, need:
 
 ---
 
+# Semantially composition with `std::unique_ptr`
```

Comment:
Typo here I think. Not sure what Semantially should be

Reply:
Updates to `Semantically`.  Let me know if you see a better wording here.